phpBB/styles/prosilver/template/viewtopic_body.html

```diff
@@ -304,6 +305,11 @@ <h3 <!-- IF postrow.S_FIRST_ROW -->class="first"<!-- ENDIF -->><!-- IF postrow.P
 					{S_FORM_TOKEN}
 				</p>
 			</form>
+			<!-- ELSE -->
+				<p class="information post-notice">
```

Comment:
post-notice should come first so information can override it.

Reply:
@hanakin could you please give an article or something about this change please?
Because [this SO answer](https://stackoverflow.com/a/3066365) as well as other ones state clearly that the order of classes defined in HTML is not important for overriding at all.